lti_consumer/templates/html/student.html

```diff
@@ -22,25 +32,35 @@ <h2 class="problem-header">
 
 % if launch_url and not hide_launch:
     % if launch_target in ['modal', 'new_window']:
-        <section class="wrapper-lti-link">
-            % if description:
-                <div class="lti-description">${description}</div>
-            % endif
-            <p class="lti-link external">
-                % if launch_target == 'modal':
-                    <button
-                        class="btn btn-pl-primary btn-base btn-lti-modal"
-                        data-target="#${element_id + '-lti-modal'}"
-                    >
-                        ${button_text or 'View resource in a modal window'} <i class="icon fa fa-external-link"></i>
-                    </button>
-                % else:
-                    <button
-                        class="btn btn-pl-primary btn-base btn-lti-new-window"
-                        data-target="${form_url}"
-                    >
-                        ${button_text or 'View resource in a new window'} <i class="icon fa fa-external-link"></i>
-                    </button>
+        <section
+            class="wrapper-lti-link"
+            aria-label="This section of the courseware is delivered by a 3rd party. As such, we cannot make any guarantees that this external resource supports the same level of accessibility."
+        >
+            % if launch_pane_html:
+                ## it is intended that authors can use HTML in the launch_pane_html parameter
+                ## so we allow this to go through unescaped
+                ${launch_pane_html | n}
+            % else:
+                % if description:
+                    ## I believe historically, we have some courses which pipe through some
+                    ## HTML in the description field, so let's preserve existing behavior
+                    <div class="lti-description">${description | n}</div>
```

Comment:
@nedbat @e0d @nasthagiri: I'm going to answer my earlier question.  I think we should be more explicit if we wish to allow HTML, and we should just use bleach with defaults as we have in other places.
@lamagnifica: We may want to mention that a limited set of HTML is acceptable for the description, but I'll leave that up to you.